src/cljs/athens/views/pages/daily_notes.cljs

```diff
@@ -92,12 +95,15 @@
       (if (empty? @note-refs)
         (dispatch [:daily-note/next (get-day)])
         (let [notes (safe-pull-many @note-refs)]
-          [:div#daily-notes (use-style daily-notes-scroll-area-style)
-           (doall
-             (for [{:keys [block/uid]} notes]
-               ^{:key uid}
-               [:<>
-                [:div (use-style daily-notes-page-style)
-                 [node-page/page [:block/uid uid]]]]))
+          [:div (use-style daily-notes-scroll-area-style)
+           ; When user scrolls to the bottom of the #daily-notes div, daily
+           ; notes from more distant past are loaded.
+           [:div#daily-notes
+            (doall
+              (for [{:keys [block/uid]} notes]
+                ^{:key uid}
+                [:<>
+                 [:div (use-style daily-notes-page-style)
+                  [node-page/page [:block/uid uid]]]]))]
```

Comment:
Why create another div?

Reply:
bump @agentydragon

Reply:
sorry about the delay.

- both the daily pages, and the "Earlier" header below, still need to be wrapped in `daily-notes-scroll-area-style`
- but in this PR I'm trying to get the reload to start earlier - so I moved the `daily-notes` ID to a container that contains *only* the daily pages, but not also the "Earlier" header. 

does the make sense?

Reply:
> 2. Delaying the loading of a new Daily Note (via debouncing, or requiring a little push beyond the scroll-to-bottom, etc.) was intended to prevent accidentally scrolling too much and creating a ton of empty Daily Notes which you had no intention of using. If that's still valuable and still function with 100ms, I'm fine with this change.

Yeah, if you just hold down `PgDn`, this PR will create pages at ~5x the speed as before, but I don't think that's a common enough problem to avoid it at the cost of slowing down page loading that much.

Reply:
I think simply changing the time should be good enough? @agentydragon